rules/google_project_iam_audit_config_invalid_member.go

```diff
@@ -0,0 +1,78 @@
+package rules
+
+import (
+	"fmt"
+
+	hcl "github.com/hashicorp/hcl/v2"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-google/project"
+)
+
+// GoogleProjectIamAuditConfigInvalidMemberRule checks whether member value is invalid
+type GoogleProjectIamAuditConfigInvalidMemberRule struct {
+	resourceType  string
+	blockName     string
+	attributeName string
+}
+
+func NewGoogleProjectIamAuditConfigInvalidMemberRule() *GoogleProjectIamAuditConfigInvalidMemberRule {
+	return &GoogleProjectIamAuditConfigInvalidMemberRule{
+		resourceType:  "google_project_iam_audit_config",
+		blockName:     "audit_log_config",
+		attributeName: "exempted_members",
+	}
+}
+
+// Name returns the rule name
+func (r *GoogleProjectIamAuditConfigInvalidMemberRule) Name() string {
+	return "google_project_iam_audit_config_invalid_member"
```

Comment:
There seems to be a problem with the consistency of the rule names. According to `docs/rules/README.md`, this name should be `google_project_iam_audit_config_invalid_member_format`. It's a good idea to give it a consistent name, whether you should give it `_format`.